### DISCUSSION TITLE
Unable to use  "http://luigi.millennium.berkeley.edu:8000/v1"

### DISCUSSION THREAD

**Discussion by shraddhasun** [ShishirPatil/gorilla#465] [General]
This is the error I'm seeing when I try to run inference_hosted.py.
Error communicating with OpenAI: HTTPConnectionPool(host='luigi.millennium.berkeley.edu', port=8000)

Is that host down?

**Comment by ShishirPatil** [comment]
Hey @shraddhasun can you try the `https` endpoint instead?

`https://luigi.millennium.berkeley.edu/v1`

Thanks!
